templates/Content/BenchmarkDotNet.BenchmarkProjectTemplate.CSharp/.template.config/template.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "identity": "BenchmarkDotNet.BenchmarkProjectTemplate.CSharp",
+  "name": "Benchmark Project",
+  "shortName": "benchmark",
+  "groupIdentity": "BenchmarkDotNet",
+  "classifications": [
+    "Benchmark",
+    "Library"
+  ],
+  "precedence": "3000",
+  "tags": {
+    "language": "C#",
+    "type": "project"
+  },
+  "description": "A project template for creating benchmarks.",
+  "author": ".NET Foundation and contributors",
+  "generatorVersions": "[1.0.0.0-*)",
+
+  "sourceName": "_BenchmarkProjectName_",
+  "preferNameDirectory": true,
+  "defaultName": "MyBenchmark",
```

Comment:
One more thing: it confuses me that currently we have "YourBenchmark" method in "MyBenchmark" class. Could we rename the class from "MyBenchmark" to "BenchmarkSuite" and "YourBenchmark" to just "Benchmark"?

Reply:
You are right - the current naming is bad and I didn't thought about it a lot when I started with this work.

I changed that in the latest PR update to this:

* The default project name is `BenchmarkSuite` (directory name is preferred).
* The default class name is `Benchmarks`.
* The default method name is `Scenario1`.

In my humble opinion, the Benchmarks are kind of "scenarios" that we try to measure and maybe compare with another one.

Let me know, if you think, that this is a good naming - as often, naming is hard.